javacomparator/DocxFile.java

```diff
@@ -0,0 +1,109 @@
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+
+/**
+ * DocxFile Class implments the methods required for comparasions of DOCX files
+ */
+public class DocxFile extends OoxmlFile {
+
+    public final String type = "docx";
+    public ArrayList<String> filesToCompare;
+    public ArrayList<String> commentToCompare;
+
+    /**
+     * Constuctor takes in the FolderPath for the file and creates a list of files to be compared.
+     * @param folderPath Folder path that has the OOXML content of the file in JSON format
+     * @param roundtripped Whether the file is roundTripped file or not.
+     */
+    public DocxFile(String folderPath, boolean roundtripped) {
+        super(folderPath, roundtripped);
+        loadFromPath();
+        filesToCompare = new ArrayList<>();
+        commentToCompare = new ArrayList<>();
+        this.filesToCompare.add("word_document.xml.json");
```

Comment:
If you need to use these types of strings, please define these as constants at the top of the class and use the constants here. 

Change these at all points in the code.